app/build.gradle

```diff
@@ -20,15 +20,16 @@ android {
 }
 
 dependencies {
-    compile fileTree(dir: 'libs', include: ['*.jar'])
+    implementation project(":androidcarrot")
+    implementation project(":contentcarrot")
+    implementation 'com.android.support:appcompat-v7:' + SUPPORT_LIBRARY_VERSION
+    implementation 'com.android.support.constraint:constraint-layout:1.0.2'
+    implementation 'com.github.dmfs.ContentPal:contentpal:' + CONTENTPAL_VERSION
+    implementation 'com.github.dmfs.ContentPal:contactspal:' + CONTENTPAL_VERSION
+
+    testCompile 'junit:junit:4.12'
```

Comment:
that should be `testImplementation` then, right?

Reply:
Yes, that's right, I missed that. Updated now, also `androidTestCompile` -> `androidTestImplementation` underneath.